src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue15151.cs

```diff
@@ -0,0 +1,81 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    public class Issue15151 : _IssuesUITest
+    {
+        public override string Issue => "15151";
+
+        public Issue15151(TestDevice device) : base(device)
+        {
+        }
+
+        [Test]
+        [Category(UITestCategories.Graphics)]
+        public void PathFBoundsShouldReturnAccurateBounds()
+        {
+            // Wait for the test page to load and elements to be displayed
+            var rectangleTightBounds = App.WaitForElement("RectangleTightBounds");
+            var rectangleFlattenedBounds = App.WaitForElement("RectangleFlattenedBounds");
+            var bezierTightBounds = App.WaitForElement("BezierTightBounds");
+            var bezierFlattenedBounds = App.WaitForElement("BezierFlattenedBounds");
+            var ovalTightBounds = App.WaitForElement("OvalTightBounds");
+            var ovalFlattenedBounds = App.WaitForElement("OvalFlattenedBounds");
+
+            // Verify the bounds elements exist and have text 
+            Assert.IsNotEmpty(rectangleTightBounds);
+            Assert.IsNotEmpty(rectangleFlattenedBounds);
+            Assert.IsNotEmpty(bezierTightBounds);
+            Assert.IsNotEmpty(bezierFlattenedBounds);
+            Assert.IsNotEmpty(ovalTightBounds);
+            Assert.IsNotEmpty(ovalFlattenedBounds);
+
+            // Get the text from the bounds labels
+            var rectTightText = App.GetText("RectangleTightBounds");
```

Comment:
This syntax is wrong. Should be something like: 
`var rectTightText = App.FindElement("RectangleTightBounds").GetText();`